### DISCUSSION TITLE
Enable Text, Number, and Table Relation on Kanban

### DISCUSSION THREAD

**Discussion by Neuxbane** [nocodb/nocodb#8703] [Features]
## Table 1
![Screen Shot 2024-06-12 at 17 19 48](https://github.com/nocodb/nocodb/assets/65763323/2fc40699-9e3f-49b7-ab26-f2e01671b056)
## Table 2
![Screen Shot 2024-06-12 at 17 20 17](https://github.com/nocodb/nocodb/assets/65763323/d5fde9b7-cd4c-4666-bd60-e54d1de3c8a0)
## The "Kanban" based on the table relation
![Screen Shot 2024-06-12 at 17 20 43](https://github.com/nocodb/nocodb/assets/65763323/cb84b8c9-4a0e-4e6b-88fd-702f94a24d25)
## But, it's not possible in here.
![Screen Shot 2024-06-12 at 17 30 11](https://github.com/nocodb/nocodb/assets/65763323/0b63e285-2d0d-4b02-bf02-8000bba64e96)
I want to add Programmers and Projects from "GitLab Users" and "GitLab Projects".

Technically, it's possible to do it on Nocodb by filtering it's relation if the relation is limited on one to many.

Please add this feature soon guys :3

**Comment by o1lab** [comment]
Have created an [issue here](https://github.com/nocodb/nocodb/issues/8704).
